crates/flux-refineck/src/refine_tree.rs

```diff
@@ -690,11 +690,21 @@ mod pretty {
         children
     }
 
+    fn pred_as_expr(pred: &Pred) -> Expr {
+        match pred {
+            Pred::Expr(expr) => expr.clone(),
+            Pred::Alias(alias_pred, args) => {
+                let func = Expr::global_func(alias_pred.name, FuncKind::Asp);
+                Expr::app(func, args.clone(), None)
+            }
+        }
+    }
```

Comment:
This is only used for printing, why not just print the `AliasPred`? We can just `preds_chain` return `(Vec<Pred>, Vec<NodePtr>)`

Reply:
Ok I'm letting this be for now, maybe we can fix if we put the assoc-pred into the expr?